kubernetes/client/ws_client.py

```diff
@@ -191,6 +191,14 @@ def run_forever(self, timeout=None):
             while self.is_open():
                 self.update(timeout=None)
 
+    def close(self, **kwargs):
```

Comment:
What are some of the variable arguments you envision being used here? Just trying to get a feel for this change...